docs/tools/terraform/reference/cookbook/kafka-karapace-recipe.rst

```diff
@@ -0,0 +1,70 @@
+Apache Kafka® with Karapace Schema Registry
+===========================================
+
+This example shows how to setup [Karapace](https://github.com/aiven/karapace) - an open source HTTP API interface and schema registry, with Aiven for Apache Kafka using `Aiven Terraform Provider <https://registry.terraform.io/providers/aiven/aiven/latest/docs>`_.
+You'll also enable the auto creation of Apache Kafka topics which will allow you to send message to topics that didn't exist already on the Apache Kafka cluster. In order to work directly with Kafka by producing and consuming messages over HTTTP, the REST API feature will be enabled. 
```

Comment:
Small typo, and maybe we can link to the https://developer.aiven.io/docs/products/kafka/howto/create-topics-automatically.html that explains topic autocreation
```suggestion
You'll also enable the auto creation of Apache Kafka topics which will allow you to send message to topics that didn't exist already on the Apache Kafka cluster. In order to work directly with Kafka by producing and consuming messages over HTTP, the REST API feature will be enabled. 
```

Reply:
Thanks; fixed.